### ISSUE TITLE
Anomaly when registering notation

### ISSUE THREAD

**Issue by RalfJung** [coq/coq#7712]
#### Version

8.8.0

#### Operating system

Linux

#### Description of the problem

The following code
```
Notation "'[tele_arg' x ';' .. ';' z ]" :=
  (TargS x ( .. (TargS z TargO) ..))
  (format "[tele_arg '[hv'  x .. z ']' ]").
```
triggers
```
Error: Anomaly "Uncaught exception Failure("List.last")."
Please report at http://coq.inria.fr/bugs/.
```

**Comment by herbelin** [comment]
Fixed by #7760 which now says `The format does not match the notation`. What location would you expect the error message to underline? `x ..`, or only `x`? (Maybe underlining `x .. z` would be better but it is more complicated to do, so I'm not proposing it.)

**Comment by RalfJung** [comment]
Between these two, I think `x ..` would be more helpful.
